samples/unit-tests/chart/options3d/demo.js

```diff
@@ -1,5 +1,32 @@
 /* eslint func-style:0 */
+QUnit.test('options3d update', function (assert) {
+    const chart = Highcharts.chart('container', {
+        chart: {
+            type: 'pie'
+        },
+        series: [{
+            data: [1, 2, 3]
+        }]
+    });
+
+    const series = chart.series[0];
+    series.points[0].graphic.isFlagged = true;
 
+    chart.update({
+        chart: {
+            options3d: {
+                enabled: true,
+                alpha: 45,
+                beta: 0
+            }
+        }
+    });
+
+    assert.ok(
+        !series.points[0].graphic.isFlagged,
```

Comment:
Just a nitpicking: could you check instead if correct elements are rendered in DOM? 3D slice should have a few sides: `top`/`side1`/`side2`/`out` (with inner radius also `inn` side).